wildmile/app/actions/CameratrapActions.js

```diff
@@ -29,8 +29,8 @@ export async function newEditCamera(req) {
 
     const camera = await Camera.create(req);
     console.log("newEditCamera camera:", camera);
-    const result = await { success: true, data: camera };
```

Comment:
@nkwsy This could be the place where you got stuck if I am guessing it correctly.


`await` typically used in when it is `Promise()`. In this case here, you don't need it. A resolved object is not a Promise. I also see you used JSON serialization and deserialization quite often. Some of it might has been handled by the framework or the lib you were using already.

Reply:
Thanks, TBH I sometimes put await's when I am having an issue and am not sure what's causing it. This works